projects/cds/package.json

```diff
@@ -15,6 +18,7 @@
   "repository": "https://github.com/SAP/cloud-commerce-spartacus-storefront",
   "peerDependencies": {
     "@angular/common": "~8.0.0",
-    "@angular/core": "~8.0.0"
+    "@angular/core": "~8.0.0",
+    "@spartacus/core": "0.1.0-alpha.6"
```

Comment:
how about adding @latest here? The intend is to use the latest anyway in our mono-repo, where we tend to publish all libs in the same time. It's a more general discussion we could have, just planting the seed here.